concordium-sdk/src/test/java/com/concordium/sdk/ClientV2GetPoolDelegatorsTest.java

```diff
@@ -0,0 +1,178 @@
+package com.concordium.sdk;
+
+import com.concordium.grpc.v2.*;
+import com.concordium.sdk.requests.BlockHashInput;
+import com.concordium.sdk.responses.accountinfo.ReduceStakeChange;
+import com.concordium.sdk.responses.accountinfo.RemoveStakeChange;
+import com.concordium.sdk.transactions.CCDAmount;
+import com.concordium.sdk.transactions.Hash;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.ByteString;
+import io.grpc.ManagedChannel;
+import io.grpc.inprocess.InProcessChannelBuilder;
+import io.grpc.inprocess.InProcessServerBuilder;
+import io.grpc.stub.StreamObserver;
+import io.grpc.testing.GrpcCleanupRule;
+import lombok.var;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.AdditionalAnswers.delegatesTo;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ClientV2GetPoolDelegatorsTest {
```

Comment:
It'd be useful to provide a short summary of the test and its intention.

Reply:
done